### ISSUE TITLE
Navigating with the keyboard

### ISSUE THREAD

**Issue by BozoDel** [Nibby/WeiqiTool#3]
Is this the right place for feature requests?

I'd like to be able to navigate through the SGF with the keyboard. Maybe down for advancing, up for going back, left and right for switching branches.

**Comment by Nibby** [comment]
Currently left and right arrow keys are for move navigation, I haven't implemented branch navigation just yet, would be cool to do though.

**Comment by BozoDel** [comment]
By currently, you mean the latest version that hasn't had an actual release yet, right?

**Comment by Nibby** [comment]
Nope, release v0.1a has the arrow key features. Though your mouse must be on the goban for the keyboard input to work.

**Comment by BozoDel** [comment]
I see!

But I think I found a bug anyway. If I press right to advance, I have to move the mouse before I can press right to advance again (or left). There's no such problem after pressing left, though.
